### ISSUE TITLE
Missing test files training NYUv2 with GT

### ISSUE THREAD

**Issue by kyk37** [LiheYoung/Depth-Anything#99]
(1)![123](https://github.com/LiheYoung/Depth-Anything/assets/112660731/7faad2b5-e404-462a-9761-48b7368e76dd)

(2) ![1234](https://github.com/LiheYoung/Depth-Anything/assets/112660731/2f6b3ef1-102e-49ae-a750-a85c920a7fbc)

I downloaded the dataset according to your [training page](https://github.com/LiheYoung/Depth-Anything/tree/main/metric_depth) which references [zoedepth dataset prep](https://github.com/cleinc/bts/tree/master/pytorch#nyu-depvh-v2), and copied it into the folder /data/nyu however the "sync" folder of (5.9Gb).
-> This folder is missing the files  referenced in [nyuv2_test_files_with_gt.txt ](https://github.com/LiheYoung/Depth-Anything/blob/main/metric_depth/train_test_inputs/nyudepthv2_test_files_with_gt.txt).

My steps. Download dataset, copy to folder. Run "python train_mono.py -m zoedepth -d nyu --pretrained_resource="" "

As you can see with the two images, I was able to get 25% of the way, however for whatever reason it is asking for files that do not exist in the sync folder. I have also looked into the files linked in [tensorflow's repo](http://datasets.lids.mit.edu/fastdepth/data/) is 32Gb and their train/val is in .h5. I also looked at the files on [huggingface](https://huggingface.co/datasets/sayakpaul/nyu_depth_v2/tree/main), and those are also .h5 without folders inside the tar files.

Also nyu_depth_v2 website on [nyu](https://cs.nyu.edu/~silberman/datasets/nyu_depth_v2.html) is broken. Every website that references it is is unable to find it.

Do you know of a location or have access to the missing jpg files, or how to parse tensorflows/huggingfaces data to match the files needed, so that I can train/fine-tune for metric_depth?

**Comment by Lilyo** [comment]
Hi, @kyk37 , 

Follow [this](https://github.com/cleinc/bts/tree/master?tab=readme-ov-file#prepare-nyu-depth-v2-test-set) to generate NYU Depth V2 test set. 

The code snippet is as follows:
```
$ cd ~/workspace/bts/utils
### Get official NYU Depth V2 split file
$ wget http://horatio.cs.nyu.edu/mit/silberman/nyu_depth_v2/nyu_depth_v2_labeled.mat
### Convert mat file to image files
$ python extract_official_train_test_set_from_mat.py nyu_depth_v2_labeled.mat splits.mat ../../dataset/nyu_depth_v2/official_splits/
```

**Comment by Zhefan-Xu** [comment]
Hi @Lilyo, thanks for your reference. I tried but it seems like "splits.mat" does not exist. Do you what is wrong?

**Comment by Lilyo** [comment]
Hi @Zhefan-Xu ,

Actually the files are placed in "bts/utils", please check [this](https://github.com/cleinc/bts/blob/master/utils/splits.mat).

**Comment by kyk37** [comment]
Thank you, I was able to load and train the model with this.
As for loading the model the training provides a .pt "dict" file which I know can be loaded as using "model.load_state_dict()".
The trained model's keys are "dict_keys(['model', 'optimizer', 'epoch'])"

Is there a reference I can use to properly load this file, and so I can infer/eval on a custom dataset? Every time I try to load the model I get "missing keys", or "incompatible keys" errors, when trying to load the dict using depth_anything-vitl14 model, or am I trying to load the wrong model?
 ~new to using checkpoint files for things


![image](https://github.com/LiheYoung/Depth-Anything/assets/112660731/c9b0bb3f-031b-4b91-b57c-3c81eaa59b2a)
